src/main/resources/upload-appeal-form/translation.json

```diff
@@ -25,12 +25,12 @@
   },
   "errors": {
     "en": {
-      "SIZE_ERROR": "File size exceeds 100Mb. Please upload a file that is less than 100Mb",
+      "SIZE_ERROR": "File size exceeds the maximum permitted value. Upload a file that is less than 100 MB",
```

Comment:
Do you think moving the common messages across various translation files to a separate file will help us avoid making the repeated changes in multiple places due to requirement change?

Reply:
In the process of making a separate ticket to keep recurrent error messages in one single file - will link here once discussed with the BA

Reply:
https://tools.hmcts.net/jira/browse/DTSSTCI-902 This is the ticket that will address this issue